dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/ServiceRegistryService.java

```diff
@@ -7,34 +7,43 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 
 import java.net.URI;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 import java.util.stream.Collectors;
 
 /**
  * A service to be registered by the service registry
  */
 public class ServiceRegistryService {
-    private final URI uri;
+    private final List<URI> uris;
     private final List<ServiceAcl> acls;
 
     public static ServiceRegistryService of(URI uri, List<com.lightbend.lagom.javadsl.api.ServiceAcl> acls) {
+        return of(Collections.singletonList(uri), acls);
+    }
+    public static ServiceRegistryService of(List<URI> uris, List<com.lightbend.lagom.javadsl.api.ServiceAcl> acls) {
```

Comment:
I really think we should change the name of this class. This look like a sandwich.

Reply:
Who doesn't like sandwiches?